src/main/scala/edu/cmu/cs/obsidian/parser/ImportProcessor.scala

```diff
@@ -12,11 +13,37 @@ object ImportProcessor {
 
     class ImportException (val message : String) extends Exception {}
 
+    def compilerPath(): Path =
+        List("TRAVIS_BUILD_DIR","OBSIDIAN_COMPILER_DIR")
+            .map(System.getenv)
+            .find(_ != null)
+            .map(Paths.get(_).toAbsolutePath)
+            .getOrElse(Paths.get("").toAbsolutePath)
+
+    def standardLibraryPath(): Path = {
+        val envPathStr = System.getenv("OBSIDIAN_STDLIB")
```

Comment:
In what cases is this necessary? I guess I'd expect the stdlib to get packaged up into the Jar file with the compiler, and then the path will always be relative to the compiler's jar file.

Reply:
If the stdlib is packaged in the jar file it won't be accessible like this anyway (we'd have to decompress the jar to access it). If it's just included alongside the jar file (at the same relative path), the default path solution still works, and if not, then the environment variable allows for flexibility in putting the standard library wherever we want.